docs/References.bib

```diff
@@ -604,6 +616,19 @@ @article{Yee1999
   url      = "http://cdsads.u-strasbg.fr/abs/1999JCoPh.150..199Y",
 }
 
+@article{Zhong2013,
+  author   = "Zhong, Xinghui and Shu, Chi-Wang",
+  title    = "A simple weighted essentially nonoscillatory limiter for
+              Runge–Kutta discontinuous {Galerkin} methods",
```

Comment:
Please also put `Runge-Kutta` in braces so it's capitalized correctly

Reply:
For my education: is `{Runge}-{Kutta}` or `{Runge-Kutta}` the preferred way of doing this?

Reply:
I don't think there is a preferred way, all caps within the `{}`s are preserved.  I've seen bitex with just the individual letters enclosed (e.g. `{R}unge-{K}utta`) or the entire string...  Looks like the existing Zhu reference does `{Runge-Kutta}` so I'd go with that...